### ISSUE TITLE
Refactor RDS handling

### ISSUE THREAD

**Issue by adrian-golian** [Blef-team/blef_game_engine#153]
Create a handler for RDS files, which will find, read, parse and validate a game state file, before returning a dataframe to whatever called for it.
Anywhere game state files are being read, there should be error handling, to return a 500 "Game state was corrupted" in case of mishaps.
Once there's a handler, we can think of mitigating issues, like repairing corrupted game states, or reading in the latest previous state.

**Comment by maciej-pomykala** [comment]
No longer relevant after #169
